contract_comm/gasprice_minimum/gasprice_minimum.go

```diff
@@ -98,17 +80,11 @@ const (
 )
 
 var (
-	gasPriceMinimumABI, _                      = abi.JSON(strings.NewReader(gasPriceMinimumABIString))
-	FallbackProposerFraction *ProposerFraction = &ProposerFraction{big.NewInt(0), big.NewInt(1)}
-	FallbackGasPriceMinimum  *big.Int          = big.NewInt(0) // gasprice min to return if contracts are not found
-	suggestionMultiplier     *big.Int          = big.NewInt(5) // The multiplier that we apply to the minimum when suggesting gas price
+	gasPriceMinimumABI, _            = abi.JSON(strings.NewReader(gasPriceMinimumABIString))
+	FallbackGasPriceMinimum *big.Int = big.NewInt(0) // gasprice min to return if contracts are not found
```

Comment:
nit: Adjust comment to `gas price minimum to return if unable to fetch from contract`

Reply:
👍